Source/AwsCommonRuntimeKit/crt/CommonRuntimeError.swift

```diff
@@ -23,6 +23,20 @@ public struct CRTError: Equatable {
         self.name = String(cString: aws_error_name(self.code))
     }
 
+    public init<T: BinaryInteger>(code: T, context: String?) {
```

Comment:
Do we still need the context string if we are generating errors from crt swift?

In the areas that are currently using context, it seems like error logging would be more appropriate. Unless we are not going to generate logs from aws-crt-swift.

Reply:
I eventually decide to keep the context, as the logging for Swift is not there yet.